aspnetcore/client-side/bower.md

```diff
@@ -14,231 +14,108 @@ uid: client-side/bower
 ---
 # Manage client-side packages with Bower
 
-<a name=bower-index></a>
+By [Rick Anderson](https://twitter.com/RickAndMSFT), [Noel Rice](http://blog.falafel.com/author/noel-rice/), and [Scott Addie](https://scottaddie.com) 
 
-By [Noel Rice](http://blog.falafel.com/author/noel-rice/), [Scott Addie](https://scottaddie.com), and [Shayne Boyer](https://twitter.com/spboyer)
+[Bower](https://bower.io/) calls itself "A package manager for the web." For .NET libraries, you still use [NuGet](https://nuget.org) package manager.
 
-Bower is a "package manager for the web." Bower lets you install and restore client-side packages, including JavaScript and CSS libraries. For example, with Bower you can install CSS files, fonts, client frameworks, and JavaScript libraries from external sources. Bower resolves dependencies and will automatically download and install all the packages you need. For example, if you configure Bower to load the Bootstrap package, the necessary jQuery package will automatically come along for the ride. For .NET libraries you still use [NuGet](https://nuget.org) package manager.
+New projects created with the ASP.NET Core project templates set up the client build process. [jQuery](http://jquery.com/) and [Bootstrap](http://getbootstrap.com/) are installed, and Bower is supported.
```

Comment:
Should the word "client" be replaced with "client-side" instead? This may add clarity.